src/Microsoft.ML.TimeSeries/AdaptiveSingularSpectrumSequenceModeler.cs

```diff
@@ -1399,8 +1395,8 @@ private void TrainCore(Single[] dataArray, int originalSeriesLength)
                     _x[i - originalSeriesLength + _windowSize] = dataArray[i];
             }
 
-            CpuAligenedMathUtils<CpuAlignedMatrixRow>.MatTimesSrc(_wTrans, _x, _y);
-            CpuAligenedMathUtils<CpuAlignedMatrixRow>.MatTranTimesSrc(_wTrans, _y, _xSmooth);
+            CpuMathUtils.MatrixTimesSource(false, _wTrans, _x, _y, _y.Length);
```

Comment:
it would be more readable to qualify what the `false` and `true` are:

`CpuMathUtils.MatrixTimesSource(transposed: false, _wTrans, _x, _y, _y.Length);`